data/serializer/src/main/java/tech/pegasys/teku/api/migrated/TotalAttestationReward.java

```diff
@@ -69,7 +69,8 @@ public TotalAttestationReward(long validatorIndex, final RewardAndPenalty reward
     this.inactivity =
         detailedRewardAndPenalty
             .getReward(RewardComponent.INACTIVITY)
-            .minus(detailedRewardAndPenalty.getPenalty(RewardComponent.INACTIVITY));
+            .minus(detailedRewardAndPenalty.getPenalty(RewardComponent.INACTIVITY))
+            .longValue();
```

Comment:
Actually, we should probably update the other calculations to have a single `.longValue()` so we only do the unboxing once. I'll update the PR.

Reply:
Actually, I introduced a bug doing it that way because we would have issues when the penalties for a component is greater than the rewards and we end up with a negative value causing a UInt underflow.